src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -59,12 +63,22 @@ private void bindListeners(ReadOnlyPerson person) {
         email.textProperty().bind(Bindings.convert(person.emailProperty()));
         person.tagProperty().addListener((observable, oldValue, newValue) -> {
             tags.getChildren().clear();
-            person.getTags().forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+            initTags(person);
         });
     }
 
+    /**
+     * Initialises tags for each person and adds corresponding style classes to each tag
```

Comment:
this is just a private helper function. do you need this detailed header comment? javadoc comments for private functions aren't as useful as it is for public functions that will be called externally